tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/core/planner.h

```diff
@@ -39,6 +39,14 @@ namespace tesseract_motion_planners
 class MotionPlanner
 {
 public:
+  /** @brief Enumeration of the types of collision checks that can be performed on a planned trajectory */
```

Comment:
Should we name this something slight different? The naming may be interpreted such that they think they are setting the collision check type used internal to the motion planner where this is controlling the post check of the trajectory.

Reply:
What do you think about `PostPlanCollisionCheckType`? I initially had something like that, but I changed it because it seemed a bit wordy.

Reply:
What about PostPlanCheckType? It would allow for us to integrated different checks other than collision if needed. If you agree I would also change the values to represent flags where you specify more than one check on the trajectory.

Reply:
For this, are you thinking an enumeration where you could combine multiple checks together with a binary `|` operator? Like this:
```
enum PostPlanCheckType
{
  TYPE_1 = 1,
  TYPE_2 = 2,
  TYPE_3 = 4,
  TYPE_4 = 8
};

// Perform checks for type 1 and type 4
PostPlanCheckType my_check = TYPE_1 | TYPE_4;
```

Reply:
That is correct.